anonymeter/stats/confidence.py

```diff
@@ -228,11 +220,12 @@ def _sanity_check(self):
                 "Attack is as good or worse as baseline model. "
                 f"Estimated rates: attack = {self.attack_rate.value}, "
                 f"baseline = {self.baseline_rate.value}. "
-                "Analysis results cannot be trusted.")
+                "Analysis results cannot be trusted.",
+                stacklevel=2,
+            )
 
         if self.control_rate is not None and self.control_rate.value == 1:
-            warnings.warn("Success of control attack is 100%. "
-                          "Cannot measure residual privacy risk.")
+            warnings.warn("Success of control attack is 100%. " "Cannot measure residual privacy risk.", stacklevel=2)
```

Comment:
```suggestion
            warnings.warn("Success of control attack is 100%. Cannot measure residual privacy risk.", stacklevel=2)
```

question: what does `stacklevel` do?

Reply:
that was the complaint from the flake8: https://github.com/PyCQA/flake8-bugbear B028. I think it makes sense to me to change it from the level 1, so users can understand where's it coming from.